src/main/java/org/gridsuite/ds/server/service/parameters/impl/ParametersServiceImpl.java

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.ds.server.service.parameters.impl;
+
+import com.powsybl.commons.config.PlatformConfig;
+import com.powsybl.dynamicsimulation.DynamicSimulationParameters;
+import com.powsybl.dynamicsimulation.json.JsonDynamicSimulationParameters;
+import com.powsybl.dynawaltz.DynaWaltzParameters;
+import org.gridsuite.ds.server.service.parameters.ParametersService;
+import org.gridsuite.ds.server.utils.xml.XmlMerge;
+import org.gridsuite.ds.server.utils.xml.implementation.SimpleXmlMerge;
+import org.springframework.stereotype.Service;
+import org.w3c.dom.Document;
+import org.xml.sax.SAXException;
+
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.transform.TransformerException;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+/**
+ * @author Thang PHAM <quyet-thang.pham at rte-france.com>
+ */
+@Service
+public class ParametersServiceImpl implements ParametersService {
+
+    @Override
+    public byte[] getEventModel() throws IOException {
+        // read the events.groovy in the "parameters" resources
+        return getClass().getResourceAsStream(Paths.get(PARAMETERS_DIR, EVENTS_GROOVY).toString()).readAllBytes();
```

Comment:
You should not use go to in intermediary Path to get a resource from class path. You just use it to format the path but keep in mind that Path.toString() does not produce the same string depending of the system (like on windows). On the other hand class path resource are always whatever the operating system is like /toto/tutu.
So in that case you should just write 
```java
getClass().getResourceAsStream(PARAMETERS_DIR + "/" + EVENTS_GROOVY))
```

Reply:
Yes! Done